docs/tempo/website/api_docs/_index.md

```diff
@@ -69,9 +69,9 @@ _For more information, please check out the official documentation of [pprof](ht
 
 ### Ingest
 
-Tempo distributor uses the OpenTelemetry Receivers as a shim to ingest trace data.
-Note that these APIs are meant to be consumed by the corresponding client SDK or a pipeline service like Grafana 
-Agent / OpenTelemetry Collector / Jaeger Agent.
+Tempo distributor uses the OpenTelemetry receiver as a foundation to ingest trace data.
```

Comment:
> receiver

Agree with the lowercase, but we really are using multiple "receivers" so I think it should remain plural. We are using the jaeger, zipkin, opencensus and otlp receivers.

Reply:
@joe-elliott Are the Jaeger, Zikpkin, OpenCensus, and OTLP receivers all part of the OpenTelemetry receiver(s)?

Reply:
Thank you. I will update the text to match.